plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/applicationexplorer/LoginWizardPage.java

```diff
@@ -119,4 +152,52 @@ protected IStatus validate() {
 			}
 		});
 	}
+	
+  private void onPasteLoginCommand(SelectionEvent e) {
+    Clipboard clipboard = new Clipboard(Display.getDefault());
+    String clipboardText = (String) clipboard.getContents(TextTransfer.getInstance());
+    if (clipboardText == null) {
+      MessageDialog.openError(getWizard().getContainer().getShell(), "Error when parsing login command",
+          "Cannot paste clipboard into the login dialog. Only text is accepted.");
+    } else if (OCCommandUtils.isValidCommand(clipboardText)) {
+      txtURL.setText(OCCommandUtils.getServer(clipboardText));
+      if (IAuthorizationContext.AUTHSCHEME_BASIC.equals(OCCommandUtils.getAuthMethod(clipboardText))) {
+        txtUsername.setText(OCCommandUtils.getUsername(clipboardText));
+        txtPassword.setText(OCCommandUtils.getPassword(clipboardText));
+      } else if (IAuthorizationContext.AUTHSCHEME_OAUTH.equals(OCCommandUtils.getAuthMethod(clipboardText))) {
+        txtToken.setText(OCCommandUtils.getServer(clipboardText));
```

Comment:
please fix issue when pasting login command: 
```suggestion
        txtToken.setText(OCCommandUtils.getToken(clipboardText));
```